recipe/conda_build_config.yaml

```diff
@@ -9,4 +9,7 @@ pin_run_as_build:
     max_pin: x.x
   scalapack:
     min_pin: x.x.x
-    max_pin: x.x
\ No newline at end of file
+    max_pin: x.x
+
+channel_sources:  # [win]
+  - conda-forge  # [win]
```

Comment:
```suggestion
```

This was only added to get flang pre-release, if we do not need it we can remove the complete block.